config/puma.rb

```diff
@@ -27,14 +27,14 @@
 # Workers do not work on JRuby or Windows (both of which do not support
 # processes).
 #
```

Comment:
Updates taken because of suggestions at https://devcenter.heroku.com/articles/deploying-rails-applications-with-the-puma-web-server